core/src/test/java/kafka/test/junit/ClusterTestExtensions.java

```diff
@@ -119,7 +131,30 @@ public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContex
         return generatedContexts.stream();
     }
 
+    @Override
+    @SuppressWarnings("booleanexpressioncomplexity")
+    public void beforeEach(ExtensionContext context) {
+        detectThreadLeak = DetectThreadLeak.of(thread -> {
+            String name = thread.getName();
+            return !name.startsWith(METRICS_METER_TICK_THREAD_PREFIX) &&
```

Comment:
Could we have a `Set` to collect those prefix and then use lambda to impl the filter? for example:
```java
    private static final Set<String> SKIPPED_THREAD_PREFIX = Collections.unmodifiableSet(Stream.of("metrics-meter-tick-thread", "scala-")
            .collect(Collectors.toSet()));
        detectThreadLeak = DetectThreadLeak.of(thread ->
            SKIPPED_THREAD_PREFIX.stream().noneMatch(prefix -> thread.getName().startsWith(prefix)));
```

Reply:
Updated it. Thanks.